magprime/__init__.py

```diff
@@ -47,6 +47,11 @@ def set_empty_donation(self):
         if not self.extra_donation:
             self.extra_donation = 0
 
```

Comment:
Ha, does this actually work?  I guess it does.  None of our other validations are defined as instance methods; we generally put them in a ``model_checks.py`` file and define them like

```python
@validation.Attendee
def some_check(attendee):  ...
```

But this does work, since the ``@validation.Attendee`` decorator takes the method before it gets turned into an instance method and then treats it as a standalone function.  I can't decide whether that's an horrible use case which just happens to work or a brilliant bit of serendipity. Probably both :P

We should probably have a built-in way to generate these checks though, so you could just say something like

```python
validation.create_money_validation(Attendee.extra_donation, 'You cannot donate negative money')
```

or something like that.  I dunno.